atd-vzv/src/views/map/Geocoder/Geocoder.js

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback } from "react";
+import Geocoder from "react-map-gl-geocoder";
+import "react-map-gl-geocoder/dist/mapbox-gl-geocoder.css";
+import { MAPBOX_TOKEN } from "../Map";
+import { geocoderBbox } from "../mapData";
+
+/**
+ * Geocoder component for the map with forwarded map ref
+ * Important: all prop objects or function must be memoized for the geocoder
+ * dropdown to close after selecting a result.
+ * @see https://github.com/SamSamskies/react-map-gl-geocoder/issues/96
+ * @param {Function} handleViewportChange - Callback function to update the map viewport
+ */
+const MapGeocoder = React.forwardRef(({ handleViewportChange }, ref) => {
+  const handleGeocoderViewportChange = useCallback(
+    (viewport) => {
+      // Speed up the flyTo transition
+      const geocoderDefaultOverrides = {
+        transitionDuration: 1500,
```

Comment:
I think maybe this might want to be even faster? I'd be comfortable with it down at `500` milliseconds or even less.

Reply:
👀 I'll leave this up to @patrickm02L since we modeled this after the speed in Moped.

Reply:
let's keep it at this speed for now.